rust/src/quic/frames.rs

```diff
@@ -557,16 +560,29 @@ impl Frame {
         if crypto_max_size > 0 && crypto_total_size == crypto_max_size {
```

Comment:
This condition means we do not handle reassembly if fragments are out of order in different packets

Reply:
Can we get an out of order event so we can see if it happens in practice?

Btw I would expect this to happen in the real world, if so no other reason due to async routing.

Reply:
Some pcap examples in https://redmine.openinfosecfoundation.org/issues/7556#change-37878

Trying to handle them in next version